### PR TITLE
Footer Component added to frontend

### PR DESCRIPTION
**Pull request by tusharjain0022**
### Solves Issue #17 
<Do not delete this template>
Description: <provide a description> Footer and contact us section added . I've also added the background color for the body tag in home.css, added google font api for "Quicksand" font and fontawesome api link in index.html which were required for the footer. Developer section in footer needs to be updated later on.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
Screenshot (372)

### CODE REVIEW

File: client/src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+/* Fontawesome icons styling */
+.iconer {
+    transition: ease 0.5s;
+}
+.iconer:hover {
+    color: #1eabd6;
+    transform: translate(0, -1px);
+}
+@media (max-width:767px){
```
Comment (EmperorYP7) [inline client/src/components/Footer/Footer.css:46]: So, everything is responsive for the footer?

File: client/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -1,12 +1,74 @@
...
+    <footer className="footer pb-0">
+        <div className="container">
+            <div className="row justify-content-center">
+                <div className="col-12 col-md-3 offset-md-1 links">
+                        <h5 className="font-weight-bold">Links</h5>
+                        <a className="  text-white"  href="https://">Home</a><br></br>
+                        <a className="  text-white"  href="https://">About</a><br></br>
+                        <a className="  text-white"  href="https://">Socities</a><br></br>
+                        <a className="  text-white"  href="https://">Projects</a><br></br>
+                        <a className="  text-white"  href="http://iiitranchi.ac.in/">IIITRanchi</a>
```
Comment (EmperorYP7) [inline client/src/components/Footer/Footer.js:38]: Suggestion: add a `target="_blank"` and `rel="noopener noreferrer"` to the anchor tag.

File: client/src/components/Footer/Footer.js [hunk 3]
```diff
@@ -1,12 +1,74 @@
...
+             <hr></hr>
+             <h3> Contact Us</h3>
+             <div className="mt-5 mb-3">
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://"><i  className="iconer fab fa-linkedin fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://github.com/houseofgeeks"><i  className=" iconer fab fa-github fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="mailto:iiitranchi@contact.info"><i  className="iconer fas fa-envelope fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.facebook.com/search/top?q=house%20of%20geeks%20-%20technical%20society%20of%20iiit%20ranchi"><i  className="iconer fab fa-facebook fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.instagram.com/hg.iiitranchi/"><i  className="iconer fab fa-instagram fa-3x"></i></a>
+                  <a className=" btn text-white" rel="noopener noreferrer" href="https://www.youtube.com/channel/UCvbh6phGLwpOnoM8g_GwlmQ"><i  className="iconer fab fa-youtube fa-3x"></i></a>
```
Comment (EmperorYP7) [inline client/src/components/Footer/Footer.js:21]: Add `target="_blank"` as well.

**Review by EmperorYP7 — APPROVED** [review]
Just look for some anchor tags. Otherwise, LGTM! :+1:

File: client/src/components/Footer/Footer.js (hunk 3 above)
Comment (tusharjain0022) [reply client/src/components/Footer/Footer.js]: Issue resolved

File: client/src/components/Footer/Footer.js (hunk 2 above)
Comment (tusharjain0022) [reply client/src/components/Footer/Footer.js]: I have added <Link> tag from react-router-dom for this. And thanks for your suggestions.

File: client/src/components/Footer/Footer.css (hunk 1 above)
Comment (tusharjain0022) [reply client/src/components/Footer/Footer.css:46]: Yes, everything looks responsive , I have tested on all screen width and on Chrome as well as on Mozilla Firefox

File: client/public/index.html [hunk 4]
```diff
@@ -9,6 +9,8 @@
       content="Web site created using create-react-app"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <script src="https://kit.fontawesome.com/90a9855ce2.js" crossorigin="anonymous"></script>
```
Comment (ankiiitraj) [inline client/public/index.html:12]: I have seen this tends to increase the rendering time. Correct me if I'm wrong.
What can be done here?
Add static svg of these icons instead

Comment (tusharjain0022) [reply client/public/index.html:12]: Yes I agree, adding static svg would decrease the rendering time , working on it !! . Thanks for the suggestion

Comment (tusharjain0022) [reply client/public/index.html]: Issue resolved : static svg icons added .

File: client/public/index.html [hunk 5]
```diff
@@ -9,6 +9,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600&display=swap" rel="stylesheet">
```
Comment (EmperorYP7) [inline client/public/index.html:12]: Maybe we should use .tff files for fonts. But it wont make its performance better, so let it be! 😛

**Review by EmperorYP7 — COMMENTED** [review]
LGTM!

File: client/public/index.html (hunk 5 above)
Comment (tusharjain0022) [reply client/public/index.html:12]: Yes, that might not make any difference in performance as same specified font files are loaded in both the cases and google font api provide all specified font files in compressed form for faster download.I might be wrong on this. 
So should we need to add .ttf/.otf files for fonts or leave it like this?

Comment (EmperorYP7) [reply client/public/index.html:12]: Nope. I dont think we should be investing time in setting up ttf without having any obvious performance boost.

**Comment by ankiiitraj** [comment]
This PR might take some time to be merged, I'm trying to pipe the front-end with netify.
Meanwhile, go for other issues.

**Comment by ankiiitraj** [comment]
Hey @tusharjain0022 make some minor commit so that netlify can create a preview build

**Comment by ankiiitraj** [comment]
Screenshot from 2020-09-11 16-30-13
Thee background don't look consistent
You can add `background-size: cover` to the background image with no-repeat

**Comment by tusharjain0022** [comment]
> Screenshot from 2020-09-11 16-30-13
> Thee background don't look consistent
> You can add `background-size: cover` to the background image with no-repeat


Issue resolved . Thanks for suggesting the changes.

**Review by ankiiitraj — APPROVED** [review]

**Comment by prskid1000** [comment]
LGM

**Pull request merged**